kobolds/biology.md

```diff
@@ -1,58 +1,66 @@
-## Physical Appearance
+# Kobold Biology
 
-### Anatomy
+1. [Anatomy](#anatomy)
+   1. [Basic Appearance](#basic-appearance)
+   1. [Scales & Coloration](#scales--coloration)
+1. [Physiology](#physiology)
+   1. [Reproduction](#reproduction)
+   1. [Gestation & Development](#gestation--development)
+
+
+## Anatomy
+
+### Basic Appearance
 
 Kobolds are a small species of people who are lizard- and bird-like. They are
-typically about 3 feet tall, with a variance of ±6 inches being common (though
+typically about 4 feet tall, with a variance of ±6 inches being common (though
 there is a rare genetic abnormality that causes some to grow to heights more in
 the 5 to 6 foot range).
 
 They have faces with pronounced snouts like many lizards. Their nostrils and
 eyes are large for their head size compared to other sentients. Eye color
-commonly can range from dark brown to golden, blue, green and yellow. They have
-no visible ears (also like lizards), though they can hear fine.
+commonly can range from dark brown to golden, blue, green and yellow. They lack
+external ears (also like lizards), though their hear is roughly comparable to
+mammalian sentients.
 
```

Comment:
I think I want to give my kobolds a dewlap like anoles have and use them for mating signaling (egg ceremony stuff???), aggression/territory and also to display emotion or something? Maybe it can change color and so may be visible even while not extended?